### PR TITLE
Bump hugo to latest version v0.121.2

### PR DESCRIPTION
**Pull request by deining**
This PR bumps hugo to its latest version v0.121.2. It also bumps hugo dependencies to their latest versions.
This PR also fixes a few deprecation warnings printed out by hugo.

- [x] Signed the [InfluxData CLA](https://www.influxdata.com/legal/cla/)
- [x] Rebased/mergeable

### CODE REVIEW

**Comment by jstirnaman** [comment]
This PR fixes the following deprecation notice:

```
docs-v2 % npx hugo --logLevel info | grep deprecate
INFO  deprecated: .Site.IsServer was deprecated in Hugo v0.120.0 and will be removed in a future release. Use hugo.IsServer instead.
```

File: README.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ including our GPG key, can be found at https://www.influxdata.com/how-to-report-
           yarn install
           ```
 
-       _**Note:** The most recent version of Hugo tested with this documentation is **0.120.4**._
+       _**Note:** The most recent version of Hugo tested with this documentation is **0.121.2**._
```
Comment (jstirnaman) [inline README.md:41]: 🥇

**Review by jstirnaman — APPROVED** [review]
LGTM! Thank you for updating these.

**Review by sanderson — APPROVED** [review]
Looks great. Thanks @deining!

**Pull request merged**